sensor_msgs/JointState state
---
bool res
